testdata/Makefile

```diff
@@ -1,16 +1,18 @@
 LLVM_PREFIX ?= /usr/bin
 CLANG ?= $(LLVM_PREFIX)/clang
-LLC ?= $(LLVM_PREFIX)/llc
 
-all: loader.elf rewrite.elf perf_output.elf invalid_map.elf
+all: loader-clang-6.0.elf loader-clang-7.elf loader-clang-8.elf rewrite.elf perf_output.elf invalid_map.elf
 
 clean:
 	-$(RM) *.elf
 
-%.elf : %.o
-	$(LLC) -march=bpf -filetype=obj -o $@ $<
+loader-%.elf: loader.c
+	$* -target bpf -O2 -g \
```

Comment:
Should this use `$(LLVM_PREFIX)`?

Reply:
The idea is to be able to do `LLVM_PREFIX=/usr/lib/llvm-8/bin` so that doesn't make much sense in this case.